api/V1/Recording.js

```diff
@@ -477,6 +478,54 @@ module.exports = (app, baseUrl) => {
     })
   );
 
+  /**
+  * @api {get} /api/v1/recordings/reprocess/:id
+  * @apiName Reprocess
+  * @apiGroup Recordings
+  * @apiParam {Number} id of recording to reprocess
+  * @apiDescription Marks a recording for reprocessing and archives existing tracks
+  *
+  * @apiUse V1UserAuthorizationHeader
+  *
+  * @apiUse V1ResponseSuccess
+  * @apiSuccess {JSON} statusCode and messages
+  */
+  app.get(
+    apiUrl + '/reprocess/:id',
+    [
+      middleware.authenticateUser,
+      param('id').isInt(),
+    ],
+    middleware.requestWrapper(async (request, response) => {
+      return await recordingUtil.reprocess(request,response)
+    })
+  );
+
+  /**
+  * @api {post} /api/v1/recordings/reprocess/multiple marks recordings for reprocessing and archives tracks
```

Comment:
Can this just be a POST to `/api/v1/recordings/reprocess`? I don't think the "multiple" part is necessary.

Reply:
Maybe i'm missing something but if I don't put a word at the end it gets confused with path:
   * @api {post} /api/v1/recordings/:devicename 
I suppose a case is that devicename could be "reprocess"

Reply:
Bummer. What if you put a trailing "/" after "reprocess" in the URL spec?

Failing that, just leave it how it is.